tests/conftest.py

```diff
@@ -156,7 +158,7 @@ def _build_jwt(webhook: PolicyPreWebhook) -> Dict[str, str]:
 
 
 @pytest.fixture(scope="session")
-def integration_config() -> MutableMapping[str, Any]:
+def integration_config():
```

Comment:
I was getting a type error here. Since it is a test and no one will use the return value directly I figured the type doesn't matter and removed it as a quick fix rather than figuring out what the correct type should be.